### ISSUE TITLE
TypeError: Cannot read property 'features' of undefined

### ISSUE THREAD

**Issue by dclipca** [thuanmb/react-mapbox-gl-cluster#11]
```
TypeError: Cannot read property 'features' of undefined
ClusterLayer._renderMarkers
node_modules/react-mapbox-gl-cluster/dist/components/ReactMapboxGlCluster/ClusterLayer.js:94
  91 |     pointStyles = _this$props2$pointSty === void 0 ? {} : _this$props2$pointSty,
  92 |     MarkerComponent = _this$props2.markerComponent;
  93 | var markerClassName = classnames('cluster-layer--point', pointClassName);
> 94 | return data.features.map(function (feature, key) {
     | ^  95 |   var coordinates = feature.geometry.coordinates,
  96 |       properties = feature.properties;
  97 |   var style = properties.style;
View compiled
ClusterLayer.render
node_modules/react-mapbox-gl-cluster/dist/components/ReactMapboxGlCluster/ClusterLayer.js:122
  119 | value: function render() {
  120 |   var clusterProps = this._getClusterProps();
  121 | 
> 122 |   return React.createElement(Cluster, Object.assign({
      | ^  123 |     ClusterMarkerFactory: this._clusterMarkerFactory
  124 |   }, clusterProps), this._renderMarkers());
  125 | }
View compiled

**Comment by thuanmb** [comment]
Seem that you're missing the required property `data`. Can you give me the example of how you are using the `ReactMapboxGlCluster`?

Did you follow the example?

```
<div className="App">
  <Map {...mapProps} onStyleLoad={this.onStyleLoad}>
    <ReactMapboxGlCluster data={data} {...this.getEventHandlers()} />
  </Map>
</div>
```

**Comment by jantonso** [comment]
@thuanmb  I'm having the same issue as @virtumonde when following the example code too. I can confirm that `data` is getting passed into the `ReactMapboxGlCluster` component, but then it is undefined in the error message above.

**Comment by thuanmb** [comment]
What is the format of your data? Can you share a minimal reproducible repo?

Edit: the `data` must be a `FeatureCollection` GeoJSON like this:
https://github.com/thuanmb/react-mapbox-gl-cluster/blob/master/src/data.js

**Comment by jantonso** [comment]
@thuanmb  Here's the gist of the code:

```javascript
import React, { Component } from 'react';
import ReactMapboxGl from 'react-mapbox-gl';
import { ReactMapboxGlCluster } from 'react-mapbox-gl-cluster';

const Map = ReactMapboxGl({
	accessToken: 'ACCESS_TOKEN_GOES_HERE',
});

const mapProps = {
	center: [-95.7129, 37.0902],
	zoom: [3],
	style: 'mapbox://styles/mapbox/streets-v8',
};

class App extends Component {

	render() {

		const data = {
			"type": "FeatureCollection",
			"features": [{
				"type": "Feature",
				"properties": {},
				"geometry": {
					"type": "Point",
					"coordinates": [
						-120.05859375,
						45.644768217751924
					]
				}
			}],
		};

		return (
			<div style={{ width: '100%', height: '100%' }}>
				<Map {...mapProps}>
					<ReactMapboxGlCluster data={data} />
				</Map>
			</div>
		);

	}

}

export default App;
```

Other information:
- `react` and `react-dom` = `v16.12.0`
- `react-mapbox-gl` = `v4.8.2`
- `react-mapbox-gl-cluster` = `v.1.4.0`

**Comment by thuanmb** [comment]
Fixed at v1.5.0. Please help to check again!

**Comment by jantonso** [comment]
Seems to have fixed it! Thanks @thuanmb
